Content.Server/Devour/DevourSystem.cs

```diff
@@ -0,0 +1,113 @@
+using Content.Server.Devour.Components;
+using Content.Shared.DoAfter;
+using Content.Shared.Mobs.Components;
+using Content.Shared.Mobs;
+using Content.Server.Popups;
+using Content.Server.Body.Systems;
+using Content.Shared.Humanoid;
+using Content.Shared.Chemistry.Components;
+using Content.Shared.Actions;
+using Robust.Shared.Containers;
+using Content.Shared.Devour;
+
+namespace Content.Server.Devour
```

Comment:
filescoped namespace, move to shared, the only thing that uses something server-only is OnDoAfter so that can stay here

Reply:
I'm confused. Could you please explain what do you mean by that? Should I split it into 2 systems, one "SharedDevourSystem", which will contain all code except OnDoAfter and the usual DevourSystem which will contain only OnDoAfter? What system should I use in the component and what to do with [Access] attribute?

Reply:
What a design preference in general? I mean, should I place everything that possible into Shared project?

Reply:
> I'm confused. Could you please explain what do you mean by that? Should I split it into 2 systems, one "SharedDevourSystem", which will contain all code except OnDoAfter and the usual DevourSystem which will contain only OnDoAfter? What system should I use in the component and what to do with [Access] attribute?

yes, shared system with a server-side inheritor for things that aren't in shared, when you move the component to shared you can point the Access to the shared system and it *should* still work for the server-side inheritor